dictionaries/software-terms/src/software-terms.txt

```diff
@@ -935,6 +935,7 @@ keyframes
 keyscan
 keyscans
 Knuth
+konami
```

Comment:
@Jason3S what would be the syntax to only allow "konami code" ?so konami would be allowed but only followed with code.

Konami is a brand that could be added to brands dictionary.

But "Konami code" is definitely a coding term.

Please note, I'm more looking for guidance from you, than requesting for a change to the author of this PR @JoshuaKGoldberg

Reply:
Since `Konami` is a company name, I think it is best to add it to [company names](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/companies/src/companies.txt) with a capitol `K`. The lower case version will be accepted in code.

It would also be ok to add it to [`gaming-terms`](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/gaming-terms/src/gaming-terms.txt) as `konami`.